kratos/includes/shared_pointers.h

```diff
@@ -55,25 +47,6 @@ unique_ptr<C> make_unique(Args &&...args) {
     return unique_ptr<C>(new C(std::forward<Args>(args)...));
 }
 
```

Comment:
@maceligueta these functions are not any longer needed since the pointer_casts are defined in intrusive_ptr.hpp

I needed to remove this since you are using variadic templates and that was interfering with the definition withing intrusive_ptr.hpp. (essentially it was not recognizing the new definition as an explicit instantiation of this)